install.sh

```diff
@@ -139,14 +139,19 @@ install_nvm_from_git() {
     fi
   fi
 
+  local remote
+  remote="$(git config --global clone.defaultRemoteName || true)"
+  if [ -z "$remote" ]; then
+      remote=origin
+  fi
```

Comment:
```suggestion
  remote="$(git config --global clone.defaultRemoteName || true) :- origin"
```

Reply:
I'm not familiar with this syntax, and can't get it to work locally. This seems to work though:

```bash
remote="$(git config --global clone.defaultRemoteName || true)"
remote="${remote:-origin}"
```
so I went with that.

Reply:
ah yes that's right, my bad :-)